### PR TITLE
Int module - addition of cmsmon-int, alertname in tag while annotating, changes in codebase in regards of it.

### PR DESCRIPTION
**Pull request by indrarahul2013**

### CODE REVIEW

File: src/go/intelligence/config.md [hunk 1]
```diff
@@ -170,7 +174,7 @@ YOU ARE FREE TO SET VALUES FOR NEW SERVICES THOUGH.
     "url": "https://monit-grafana.cern.ch",
     "dashboardSearchAPI": "/api/search",
     "annotationAPI": "/api/annotations",
-    * "tags": ["cmsweb-play"],
+    * "tags": ["cmsweb", "das", "jobs", "prod"],
```
Comment (vkuznet) [inline src/go/intelligence/config.md:177]: please remove das since das is part of cmweb as well

File: src/go/intelligence/config.md [hunk 2]
```diff
@@ -213,9 +217,20 @@ YOU ARE FREE TO SET VALUES FOR NEW SERVICES THOUGH.
...
       "annotationMap": {
-        # "label": "shortDescription",              #DO NOT CHANGE IT, UNIQUE FOR SSB
-        "actions": ["intervention"],
-        "systems": ["network", "database", "db"],
+        "label": "shortDescription",               #DO NOT CHANGE IT, UNIQUE FOR SSB
+        "annotations": [
+          {
+            "actions": ["intervention", "outage"],
+            "systems": ["network", "database", "db"],
+            "tags": ["cmsweb", "das"]
```
Comment (vkuznet) [inline src/go/intelligence/config.md:225]: please remove das as it part of cmsweb

File: src/go/intelligence/config.md [hunk 3]
```diff
@@ -213,9 +217,20 @@ YOU ARE FREE TO SET VALUES FOR NEW SERVICES THOUGH.
...
+          },
+
+          {
+            "actions": ["update", "upgrade"],
+            "systems": ["network", "database", "db"],
+            "tags": ["jobs"]
```
Comment (vkuznet) [inline src/go/intelligence/config.md:231]: I saw above tags with prod and jobs, should we have them here too?

Comment (indrarahul2013) [reply src/go/intelligence/config.md:231]: Not actually. This is just an example. Giving an overview of the config file.

File: src/go/intelligence/config.md (hunk 1 above)
Comment (indrarahul2013) [reply src/go/intelligence/config.md:177]: Done !

File: src/go/intelligence/config.md (hunk 2 above)
Comment (indrarahul2013) [reply src/go/intelligence/config.md:225]: done !

**Pull request merged**